README.md

```diff
@@ -25,13 +25,13 @@ Calling `withLock` with the same `scope` and `key` will ensure that the callback
 ```typescript
 import {withLock} from "lifecycle-utils";
 
-const scope = {}; // can be a reference to any object you like
+const scope = {name: 'Tommy'}; // can be a reference to any object you like
```

Comment:
I think it's be better to create another code example for the usage of `this`.
Also, update the console log output comment below to match the updated example